### ISSUE TITLE
Date Range - Manual entry broken ?

### ISSUE THREAD

**Issue by bagintz** [uxsolutions/bootstrap-datepicker#721]
To duplicate issue:
1) go to the demo (http://eternicode.github.io/bootstrap-datepicker/)
2) choose "range"
3) enter "11/10/2012" into first date
4) press tab
5) enter "11/10/2012" into second date

issue: when you get to the start of the year for the second date, it starts to change the year in the first date too.  As far as I know, it does not actually change the date that is stored, but the UX shows "11/10/2" for the first date.

**Comment by chappoo** [comment]
Agreed this is an issue for us too. Workaround at the moment is to specify a startDate of 01/01/1900 (or as appropriate).

http://bootstrap-datepicker.readthedocs.org/en/latest/options.html#startdate

**Comment by stanpalatnik** [comment]
Are there any plans to address this?

**Comment by acrobat** [comment]
I hope we can fix this in 1.7.X!

**Comment by acrobat** [comment]
There is an fix proposed for this issue in PR #1891. Can you guys verify if this fixes the issue?

**Comment by amucino** [comment]
Is there a workaround yet ?

**Comment by btn-ninja** [comment]
@chappoo Can you please explain how to do the workaround?  
Docs were unclear and this does not work:

`<input type="text" class="jDatepicker" data-date-start-date="01/01/1970">`

**Comment by chappoo** [comment]
@btn-ninja it's been a while, but that's pretty much what I was doing, setting the data-date-start-date attr (running v1.6.0).  It could be that the workaround is no longer valid on more recent versions, something I haven't tested.

**Comment by btn-ninja** [comment]
Thanks!  It appears to be fixed in 1.7.x, so I will need to update my version (and get my team to run extensive testing).  

Correctly working example using version 1.7.1 (link for future googlers):  
https://codepen.io/btn-ninja/pen/QMZKBK
